docs/puzzles/day21.md

```diff
@@ -1,11 +1,312 @@
 import Solver from "../../../../website/src/components/Solver.js"
 
-# Day 21
+# Day 21: Dirac Dice
+by @sjrd
 
 ## Puzzle description
 
 https://adventofcode.com/2021/day/21
 
+## Modeling and parsing the input
+
+There are no more surprises here, so let us directly see the code:
```

Comment:
It's okay not to give much explanations but we should not pretend it is obvious for everyone. Someone may start here without knowing about previous solutions.

```suggestion
As we did for previous solution, we start by modelling the problem and parsing the input.
```